frontend/src/pages/trucks/components/add-truck-form/libs/helpers/get-truck-entity.helper.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  type TruckEntity,
+  type TruckFormModel,
+} from '~/packages/trucks/libs/types/types.js';
+
+function getTruckEntity(payload: TruckFormModel): Omit<TruckEntity, 'id'> {
+  const { ...truckDetails } = payload;
+
+  return {
+    ...truckDetails,
+    manufacturer: truckDetails.manufacturer.value,
+    towType: truckDetails.towType.value,
+  };
```

Comment:
do you need truckDetails new variable ? why not to use payload
also I think we prefer arrow functions instead function declaration